aspnetcore/mvc/views/tag-helpers/built-in/anchor-tag-helper.md

```diff
@@ -262,9 +262,9 @@ The generated HTML:
 
 ### asp-page
 
-The [asp-page](xref:Microsoft.AspNetCore.Mvc.TagHelpers.AnchorTagHelper.Page%2A) attribute is used with Razor Pages. Use it to set an anchor tag's `href` attribute value to a specific page. Prefixing the page name with a forward slash ("/") creates the URL.
+The [asp-page](xref:Microsoft.AspNetCore.Mvc.TagHelpers.AnchorTagHelper.Page*) attribute is used with Razor Pages. Use it to set an anchor tag's `href` attribute value to a specific page. Prefixing the page name with `/` creates a URL for a matching page from the root of the app:
```

Comment:
Consider the attribute `asp-page="/Test/Attendees"`.  The _page name_ is `Attendees`.  While what you have written is technically true, it is not helpful.